### ISSUE TITLE
dot-notation: false positive when using accents

### ISSUE THREAD

**Issue by christophehurpeau** [oxc-project/tsgolint#828]
Hello,

Thank you for this awesome tool that is super fast compared to typescript-eslint.

Here an example of false-positive:

```
  × typescript-eslint(dot-notation): ["Prénom"] is better written in dot notation.

 142 │               firstName: validatedValues["Prénom"],
     ·                                          ────────
 143 │               lastName: validatedValues.Nom,
     ╰────

```

**Comment by camc314** [comment]
Thanks for the report - 0.18.0 is releasing now with a fix.
